src/commands/basemaps-github/create-pr.ts

```diff
@@ -0,0 +1,64 @@
+import { boolean, command, flag, option, optional, string } from 'cmd-ts';
+import { logger } from '../../log.js';
+import { verbose } from '../common.js';
+import { CliInfo } from '../../cli.info.js';
+import { Category, MakeCogGithub, parseCategory } from './make.cog.github.js';
+import { ConfigLayer, standardizeLayerName } from '@basemaps/config';
+
+export const CommandListArgs = {
+  verbose,
+  layer: option({
+    type: string,
+    long: 'layer',
+    description: 'Input config layer import into basemaps-config',
+  }),
+  category: option({
+    type: optional(string),
```

Comment:
did we want to be more specific `new Set(['Rural Aerial Photos', ...])` ?

Reply:
I think we can put any category for this? Like if we have new scanned imagery, we should be able to import as new category here?